homeassistant/components/flo/__init__.py

```diff
@@ -30,10 +31,9 @@ async def async_setup(hass: HomeAssistant, config: dict):
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     """Set up flo from a config entry."""
-    hass.data[DOMAIN][entry.entry_id] = {}
     session = async_get_clientsession(hass)
     try:
-        hass.data[DOMAIN][entry.entry_id]["client"] = client = await async_get_api(
+        hass.data[DOMAIN][CLIENT] = client = await async_get_api(
```

Comment:
This means each config entry will override this value ?

Reply:
Good point